lib/Enum/TextTruncate.lua

```diff
@@ -0,0 +1,6 @@
+local createEnum = import("../createEnum")
+
+return createEnum("TextTruncate", {
+    None = 0,
+    AtEnd = 1,
```

Comment:
This file is indented with spaces -- the standard for Lemur and Roblox codebases is tabs!

You can check out our style guide (https://roblox.github.io/lua-style-guide/) for other details. We opened it up to the public super recently but haven't quite announced it yet.